invenio_circulation/api.py

```diff
@@ -34,8 +34,9 @@ def create(cls, data, id_=None, **kwargs):
         """Create Loan record."""
         data["$schema"] = current_jsonschemas.path_to_url(cls._schema)
         ref_builder = current_app.config.get("CIRCULATION_ITEM_REF_BUILDER")
-        item_attached = data.get("item_pid") and data["item_pid"] != ""
-        if ref_builder and item_attached:
+        item_pid = data.get("item_pid")
```

Comment:
I think this like was specifically checking for `None` or empty string `''`.
Are you sure we want to remove it?

Reply:
@ntarocco If I understood correctly,
the statement `if item_pid: ` will fail in both cases `None` or `''`, so I thought it was an extra unnecessary check.

If it doesn't break anything I don't see why not remove it.

Reply:
Do we still set `item_pid` to empty string in the UI when item is not attached? That's we were checking explicitely here I think.

Reply:
@zzacharo Yes the `item_pid` and is an empty string if there is no item attached